raiden/network/rpc/client.py

```diff
@@ -443,88 +481,164 @@ def monkey_patch_web3(web3: Web3, gas_price_strategy: Callable) -> None:
     Eth.call = patched_web3_eth_call
 
 
-class TransactionSlot:
```

Comment:
I'm forcing the creation of the slot to go through the client by hiding the implementation. This prevents using nonces wrongly.